project/HmrcBuild.scala

```diff
@@ -50,6 +51,7 @@ private object BuildDependencies {
     val pegdown = "org.pegdown" % "pegdown" % "1.5.0" % scope
     val playTest = "com.typesafe.play" %% "play-test" % PlayVersion.current % scope
     val httpVerbsTest = "uk.gov.hmrc" %% "http-verbs-test" % "0.1.0" % scope
+    val mockito = "org.mockito" % "mockito-core" % "1.9.0" % scope
```

Comment:
Formerly, mockito was acquired transitively via Play 2.3. With 2.5, Play no longer depends on mockito so it needs to be explicitly declared.